dockerfiles/reverse-proxy/README.md

```diff
@@ -0,0 +1,13 @@
+# reverse-proxy
+
+The reverse-proxy Dockerfile is the Barcelona supported reverse proxy (based on nginx) that is configured to cleanly integrate with the other Barcelona images.
```

Comment:
Apologies if this is incorrect, I guessed a little 😄 . If there's a better description for the image let me know and I'll update the PR 😄